contrib/admin/quay_container_cleanup.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
+"""
+Script to scan and delete unnecessary (non-Toil Release) quay.io containers/tags.
+
+Long tag names in 'ucsc_cgl/toil' with the 'a1-' denoting a dev version will be printed as a list and
+the user asked to confirm their deletion.
+
+NOTE TO USER: Try not to delete images for currently running tests.  Check CI/CD first before running.
+"""
+import requests
+import os
+
+
+# more of a comment; this list really doesn't need to be updated since we filter by length.
+whitelist = [
```

Comment:
This name could be a bit more descriptive, and I think the semantics are backward. These are the ones we always keep, and never delete, so we should call this `ALWAYS_KEEP` or `IGNORED_TAGS` or `BLOCKLIST` or something. The way it is now it looks a bit like these are the ones that the deleter is allowed to delete.